src/components/metabase/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { Environment } from "@kosko/env";
+import { merge } from "@socialgouv/kosko-charts/utils/@kosko/env/merge";
+
+import type { DeploymentParams } from "../../utils/createDeployment";
+import type { AppConfig } from "../app";
+import { create as createApp } from "../app";
+
+type CreateResult = unknown[];
+
+export const create = (
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  {
+    env,
+    config = {},
+    deployment = {},
+  }: {
+    env: Environment;
+    config?: Partial<AppConfig>;
+    deployment?: Partial<DeploymentParams>;
+  }
+): CreateResult =>
+  createApp("metabase", {
+    config: merge(
+      {
+        containerPort: 3000,
+        image: "metabase/metabase:v0.37.4",
```

Comment:
Might want to use a renovate friendly version code

Reply:
good ?